client/ayon_core/pipeline/workfile/workfile_template_builder.py

```diff
@@ -853,11 +844,10 @@ def get_template_preset(self):
             keep_placeholder = True
 
         if not path:
-            raise TemplateLoadFailed((
-                "Template path is not set.\n"
-                "Path need to be set in {}\\Template Workfile Build "
-                "Settings\\Profiles"
-            ).format(host_name.title()))
+            self.log.info(
+                "Template path is not set."
+            )
+            return
```

Comment:
@MustafaJafar I believe this path of entry would still error - because then the function returns `None` so usage of `get_template_preset` call result in `build_template` will still error. I'll need to still patch that up.

Reply:
Hmm - not sure what the "best" fallback behavior would be. With the current logic it just seems weird if it'd still continue to save since I'd need to check whether an actual filepath was returned, etc. I'll need to think about this some more :')